crafters/nlp/PDFExtractorSegmenter/tests/test_pdf_flow.py

```diff
@@ -0,0 +1,11 @@
+from jina.flow import Flow
+import os
+
+def test_pdf_flow():
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    path = os.path.join(cur_dir, 'cats_are_awesome_text.pdf')
+
+    f = Flow().add(uses='PDFExtractorSegmenter')
+    with f:
+        f.index_files([path], output_fn=print)
```

Comment:
output_fn should be a validation function where you can assert how the returned document looks like